src/core/UserIconGroup/UserIconGroup.tsx

```diff
@@ -0,0 +1,146 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import React from 'react';
+import { useTheme } from '../utils/hooks/useTheme';
+import cx from 'classnames';
+import { CommonProps } from '../utils/props';
+
+export type UserIconGroupProps = {
+  /**
+   * UserIconGroup max number of icons unstacked
+   * @default 5
+   */
+  maxIcons?: number;
+  /**
+   * UserIconGroup is group needs to be stacked
+   * @default 'true'
+   */
+  stacked?: boolean;
+  /**
+   * UserIconGroup type
+   * @default 'false'
+   */
+  animated?: boolean;
+  /**
+   * UserIconGroup type
```

Comment:
Its not type.. Size more, I guess..

Reply:
interesting about the size too.. Cause user needs to pass size prop to the group component and also hopefully same in every UserIcon component...

Reply:
oh, I see you just override all user icons with the container size.. Might mention that in comment too.